cmd/expandybird/expander/expander_test.go

```diff
@@ -756,44 +756,16 @@ func TestReplicatedService(t *testing.T) {
 		&expansion.ServiceResponse{
 			Resources: []interface{}{
 				map[string]interface{}{
```

Comment:
Yes. Worth exploring. This restructuring is fragile at best, but it works and for now, that's what we needed.